src/registry.rs

```diff
@@ -29,13 +30,23 @@ use crate::Span;
 #[builder(default)]
 pub struct Config {
     /// Whether to include the **verbose** span in the await-tree.
-    verbose: bool,
+    pub verbose: bool,
```

Comment:
We may keep these fields private (or `pub(crate)`) and use `ConfigBuilder` to construct a `Config`.

Reply:
> We may keep these fields private (or `pub(crate)`) and use `ConfigBuilder` to construct a `Config`.

Current config is simple, only three fields, but the disign pattern is up to you. Maybe the current version is enough.

Reply:
We already have `ConfigBuilder` now (by deriving `Builder`) so there's no extra effort required. :)